packages/documentation/src/stories/foundation/typography/typography.components.tsx

```diff
@@ -0,0 +1,400 @@
+import React from 'react';
+import { objectify } from '../../../utils/sass-export.ts';
+import { forEach } from '../../../utils/react.ts';
+import scss from './typography.components.scss';
+
+const SCSS_VARIABLES = objectify(scss);
+const baseFontSize = parseFloat(SCSS_VARIABLES.base.fontSize);
+
+export function FontFaceWrapper (props: { children: React.ReactElement[] | null }) {
+  return <div className="sb-fontface-wrapper">
+    { props.children }
+  </div>;
+};
+
+export function FontFace (props: { face: string, family: string, weight: string, style: string }) {
+  return <article className="sb-fontface">
+    <div className={ `sb-fontface__typo font-${props.family} fw-${props.weight} fst-${props.style}` }>
+      Aa
+    </div>
+
+    <div className={ `sb-fontface__chars font-${props.family} fw-${props.weight} fst-${props.style}` }>
+      ABCDEFGHIJKLMNOPQRSTUVWXYZ<br/>
+      abcdefghijklmnopqrstuvwxyz<br/>
+      1234567890(,.;:?!$&*)
+    </div>
+
+    <div className="sb-fontface__settings">
+      <div className="settings__face">{ props.face }</div>
+      <div className="settings__weight">
+        <span>Weight:</span> { props.weight }
+      </div>
+      <div className="settings__style">
+        <span>Style:</span> { props.style }
+      </div>
+    </div>
+  </article>;
+};
+
+export function FontSizesAndLineheights () {
+  return <div className="sb-fontsizes">
+    <table className="table table-striped table-bordered">
+      <thead>
+        <tr>
+          <th className="w-25">Name</th>
+          <th>Font Size</th>
+          <th>Line Height</th>
```

Comment:
I'm not sure if we're talking about the same thing here...
It's not about class-names or the design-system `sizes` here, even if they're seem to be very similar.
This list shows the defined font-sizes and their mapped line-heights in the type.scss file.

Therefore I think it is important to list these informations somewhere for our devs.

Reply:
I'm not even sure if we provide this class in the moment. I just added it, because there are some line-height classes provided by bootstrap (see https://getbootstrap.com/docs/5.3/utilities/text/#line-height).

Listing class names and sass variables is a good idea!

Reply:
I would leave the line-height information as it is for the sake of completeness.